### PR TITLE
memesuite: update to version 5.5.0

### PR DESCRIPTION
**Pull request by tlawbailey**
* update to version 5.5.0
* bed2fasta: new command for creating sequence FASTA files from BED files
* update-sequence-db: enhancements for easier downloading of public sequence databases
* fix bugs in STREME, MOMO, MEME, FIMO, GLAM2 and AMA

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.6.8 20G730 x86_64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?

### CODE REVIEW

File: science/memesuite/Portfile [hunk 1]
```diff
@@ -3,48 +3,72 @@
...
+
+# Make parallel version of MEME using MacPorts port of openmpi.
+variant parallel description {Build MEME for parallel mode.} {
+                    depends_lib-append port:openmpi
+                    configure.args-append --with-mpidir=${prefix}
+}
+
+notes "The MEME Suite is free for educational, research and non-profit purposes. Commercial users should obtain a license as described here: https://meme-suite.org/meme/doc/copyright.html."
+
 use_parallel_build  yes
```
Comment (reneeotten) [inline science/memesuite/Portfile:69]: this is the default and can thus be removed

File: science/memesuite/Portfile [hunk 2]
```diff
@@ -3,48 +3,72 @@
...
-                    --with-python=${prefix}/bin/python3.7 \
-                    --with-perl=${prefix}/bin/perl5.28 \
+                    --with-python=${prefix}/bin/python3.10 \
+                    --with-perl=${prefix}/bin/perl${perl_vers} \
                     --with-gs=${prefix}/bin/gs
 
+configure.cppflags-delete    -I${prefix}/include
+
+#
+#
```
Comment (reneeotten) [inline science/memesuite/Portfile:55]: no need for two commented out lines

**Review by reneeotten — COMMENTED** [review]
thanks @tlawbailey a few minor comments for your consideration

File: science/memesuite/Portfile (hunk 1 above)
Comment (tlawbailey) [reply science/memesuite/Portfile:69]: I agree to this change.

File: science/memesuite/Portfile (hunk 2 above)
Comment (tlawbailey) [reply science/memesuite/Portfile:55]: I agree to this change.

**Comment by reneeotten** [comment]
@tlawbailey I am glad you agree with my comments 😉. Please make the  requested changes and push to this PR so we dan merge it.

**Comment by reneeotten** [comment]
@tlawbailey you did not remove the `use_parallel_build  yes` yet. Additionally, is there a reason to build a vendored version of libxml2 and libxslt; we prefer to use the available ports for these packages in MacPorts. Can that be done?

**Comment by reneeotten** [comment]
thanks @tlawbailey I was playing around this PR and resolving the conflict and besides pushing to this PR I also pushed it to `master` so it's already in (see 187b5b6853d05da8a4c90b3c518d3a90169257e1). Therefore closing this PR - thanks for the update!

**Pull request closed without merging**